changelog.md

```diff
@@ -34,8 +34,8 @@
 * UI fix to bottom margin in conversation view (#1391)
 * (Dev) Refactors that break up the widgets module into smaller components (#1377-1383, #1390, #1393, #1394)
 * (Dev) Use Debian Stable container images in CI (#1385)
-* (Dev) New templates for creating different types of Github Issues (#1392)
-* (Dev) Add localization testing task to the release Github Issue template (#1401)
+* (Dev) New templates for creating different types of GitHub Issues (#1392)
+* (Dev) Add localization testing task to the release GitHub Issue template (#1401)
```

Comment:
brand

Reply:
@rocodes @cfm How do you feel about fixing typos in the change log? (I'm absolutely fine with that, asking just in case.)

Reply:
This is easy to revert is needed, anyway.